tests/dummy/app/app.js

```diff
@@ -3,7 +3,7 @@ import Resolver from 'ember/resolver';
 import loadInitializers from 'ember/load-initializers';
 import config from './config/environment';
 
-var App;
+let App;
 
 Ember.MODEL_FACTORY_INJECTIONS = true;
 
```

Comment:
The rest of this file should following spacing and formatting standards.  Change one change them all :-)